orally-network/sybil
# Multirpc requests implemented + rpc requests approach changed

In this PR:
1. Fixed Makefile config for successful local build
2. Domain authentication logic was removed from the code. 
3. Multirpc requests implemented, using direct evm-rpc methods(e.g. `eth_call`) instead of abstract `request`
4. Fallback approach implemented for all evm-rpc methods. First call - with default providers, second(fallback) call - with sybil custom ones 
5. Cargo warnings fixed. WIP

How to test domain logic removal:
- deploy canister locally and prepare canister for usage(api_key generation, balance deposit..)
- open method request link in a browser(so no domains passed as with curl request)
- you should see method output instead of "domain not found" error or code panic
